conda/common/path.py

```diff
@@ -88,3 +88,41 @@ def win_path_ok(path):
 
 def win_path_double_escape(path):
     return path.replace('\\', '\\\\') if on_win else path
+
+
+def maybe_pad(name, pad="_"):
+    new_name = name if name.startswith(pad) else "%s%s" % (pad, name)
+    new_name = new_name if name.endswith(pad) else "%s%s" % (new_name, pad)
+    return new_name
```

Comment:
Too much code.  How about

    return '%s%s%s' % (pad, name.strip(pad), pad)

Could also call it `ensure_pad`.  Probably more accurate.